src/Servers/Kestrel/Core/src/Internal/Http3/Http3Stream.cs

```diff
@@ -159,7 +159,9 @@ public void OnStaticIndexedHeader(int index, ReadOnlySpan<byte> value)
             OnHeader(knownHeader.Name, value);
         }
 
-        public override void OnHeader(ReadOnlySpan<byte> name, ReadOnlySpan<byte> value)
+        public void OnHeader(ReadOnlySpan<byte> name, ReadOnlySpan<byte> value) => OnHeader(name, value, true);
+
+        public override void OnHeader(ReadOnlySpan<byte> name, ReadOnlySpan<byte> value, bool checkForNewlineChars = true)
```

Comment:
why is there a default parameter here? This should only be called from the other OnHeader overload?